code/API_definitions/check-device-connectivity.yaml

```diff
@@ -0,0 +1,245 @@
+openapi: 3.0.3
+info:
+  title: Check device connectivity API
+  description: Service Enabling Network Function API for monitoring device connectivity
+  termsOfService: http://swagger.io/terms/
+  contact:
+    email: project-email@sample.com
+  license:
+    name: Apache 2.0
+    url: https://www.apache.org/licenses/LICENSE-2.0.html
+  version: 0.2.0
+externalDocs:
+  description: Product documentation at CAMARA
+  url: https://github.com/camaraproject/
+security:
+  - {}
```

Comment:
My preference would be to remove the "no authentication" option, or at least comment that it is only for test / development purposes, but it can stand for now

Reply:
Just a small detail but for consistency sake we should have the same security clause for all Camara API specificationr ?
the QoD API v0.8.0 has:
`security:
  - oAuth2ClientCredentials: []`

Perhaps we should aligh QoD security.